src/Raven.Server/Smuggler/Documents/Actions/DatabaseIndexActions.cs

```diff
@@ -26,8 +30,15 @@ public DatabaseIndexActions([NotNull] AbstractIndexCreateController controller,
             _batch = _controller.CreateIndexBatch();
     }
 
-    public async ValueTask WriteIndexAsync(IndexDefinitionBaseServerSide indexDefinition, IndexType indexType)
+    public async ValueTask WriteIndexAsync(IndexDefinitionBaseServerSide indexDefinition, IndexType indexType, AuthorizationStatus authorizationStatus)
     {
+        if (_configuration.Security.AuthenticationEnabled &&
+            authorizationStatus == AuthorizationStatus.ValidUser)
+        {
+            throw new CertificateException(
```

Comment:
Don't we need to distinguish between C# indexes and JS indexes?

Reply:
We are getting here only with Auto indexes

Reply:
Can we rename the method to `WriteAutoIndexAsync` then?

Reply:
Done